lib/components/molecules/Accordion/AccordionHeader/tests/AccordionHeader.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { AccordionHeaderVanilla } from '../index';
+
+describe('<AccordionHeader />', () => {
+  let AccordionHeaderComponent = '';
+  beforeEach(() => {
+    AccordionHeaderComponent = shallow(<AccordionHeaderVanilla>Test</AccordionHeaderVanilla>);
+  });
+
+  test('should render', () => {
```

Comment:
This is not enough, where do you test the if your content is being placed properly

Reply:
Added